src/main/java/uk/gov/hmcts/ccd/domain/service/search/filter/CaseStateFilter.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.ccd.domain.service.search.filter;
+
+import java.util.List;
+
+import static uk.gov.hmcts.ccd.domain.service.common.AccessControlService.CAN_READ;
+
+import uk.gov.hmcts.ccd.domain.service.common.AuthorisedCaseDefinitionDataService;
+
+public class CaseStateFilter {
```

Comment:
same as the other comment, I'd merge this into `ElasticsearchCaseStateFilterFactory`

Reply:
merged